modules/next-backend/logic/getProjectCreation.ts

```diff
@@ -0,0 +1,55 @@
+import { Sql } from "../db";
+
+import {
+  LovelaceAmount,
+  ProjectId,
+  UnixTimestamp,
+} from "@/modules/business-types";
+
+type Params = {
+  projectId: ProjectId;
+};
+
+// Add optional field for error.
+export type CreationActivity$Response =
+  | {
+      error: null;
+      createdAt: UnixTimestamp;
+      sponsorshipAmount: LovelaceAmount | null; // undefined means no sponsor
+    }
+  | {
+      error: "no-project-found";
+    };
+
+export async function getProjectCreation(
```

Comment:
```suggestion
export async function getProjectCreationActivity(
```
Rename this file too.
And don't forget to rebase.